Packs/AzureActiveDirectory/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Azure Active Directory Identity and Access",
     "description": "Use Microsoft Graph to query information from Azure Active Directory",
     "support": "xsoar",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
```

Comment:
I feel this warrants a minor bump.

Reply:
Correct. https://github.com/demisto/content/pull/14352/commits/2d382d8fa7ab57dd26e56b3927b83764de459c26